crates/bevy_asset/src/loader.rs

```diff
@@ -307,6 +308,271 @@ pub enum DeserializeMetaError {
     DeserializeMinimal(SpannedError),
 }
 
+/// A builder for loading nested assets inside a `LoadContext`.
```

Comment:
```suggestion
/// A builder for loading nested assets inside a [`LoadContext`].
```
This should be a proper doc link.